app/views/layouts/application.html.slim

```diff
@@ -51,4 +51,36 @@
       = render partial: 'shared/session_prompt'
       = yield
 
+- content_for :cookie_message do
+  - unless cookie_settings['seen']
+    .govuk-cookie-banner.confirmation aria-label=("Cookies on [name of service]") role="region"
+      .govuk-cookie-banner__message.govuk-width-container
+        .govuk-grid-row
+          .govuk-grid-column-two-thirds
+            h2.govuk-cookie-banner__heading.govuk-heading-m.bold= t('banner.title')
+            .govuk-cookie-banner__content
+              p= t('banner.first_line')
+              p= t('banner.second_line')
+          .govuk-button-group
+            = simple_form_for cookie_form, as: :cookie, url: cookies_path, method: :post, html: {style: 'display: inline'} do |f|
+              = f.hidden_field :usage, value: 'true'
+              = f.hidden_field :return_path, value: path_only(request.url)
+              = f.button :submit, t('banner.accept_analytics')
+            = simple_form_for cookie_form, as: :cookie, url: cookies_path, method: :post, html: {style: 'display: inline'} do |f|
+              = f.hidden_field :usage, value: 'false'
+              = f.hidden_field :return_path, value: path_only(request.url)
+              = f.button :submit, t('banner.reject_analytics')
+              = link_to t('banner.cookie_link'), edit_cookies_path
```

Comment:
The cookies link does not need to be in the form - please un indent 1 level to bring it out